android/app/src/main/kotlin/com/cBreez/client/job/LnurlPayInfo.kt

```diff
@@ -58,16 +59,16 @@ class LnurlPayInfoJob(
                 Constants.NOTIFICATION_CHANNEL_LNURL_PAY,
                 context.getString(if (success) R.string.lnurl_pay_info_notification_title else R.string.lnurl_pay_notification_failure_title),
             )
-        } catch (e: SdkException) {
+        } catch (e: Exception) {
             Logger.tag(TAG).warn { "Failed to process lnurl: ${e.message}" }
-            fail(e.message, request.replyURL)
+            if (request != null) {
+                fail(e.message, request.replyURL)
+            }
             notifyChannel(
                 context,
                 Constants.NOTIFICATION_CHANNEL_LNURL_PAY,
                 context.getString(R.string.lnurl_pay_notification_failure_title),
             )
         }
-
-        fgService.shutdown()
```

Comment:
Why was this removed? Is it because `LnurlPayInvoice` task is expected to start soon afterwards?

What if we don't have enough inbound liquidity to pay the lnurl invoice? I don't think we do receive a _"lnurlpay_invoice"_ notification for rejected/failed cases.

Reply:
My bad, sorry. Reverted the change